### ISSUE TITLE
The training time is long and the results are low

### ISSUE THREAD

**Issue by 1301358882** [Leon1207/3DGCTR#4]
Using an A6000 card for joint training on 3DVG and 3DDC, with a batch size of 8, it has taken nearly a week to reach only 26 epochs, and the current best CIDEr@0.5 is 100075. Is this normal?
![微信图片_20241226131923](https://github.com/user-attachments/assets/cd89ad8a-cc61-4606-9f8d-e4f448085765)

**Comment by Leon1207** [comment]
Yes, you can try using a test script to test the current checkpoint to get the normal results.

**Comment by 1301358882** [comment]
> Yes, you can try using a test script to test the current checkpoint to get the normal results.

When I was testing, I got an error .
![Uploading 1735524339830.jpg…]()

**Comment by 1301358882** [comment]
> Yes, you can try using a test script to test the current checkpoint to get the normal results.

As shown in Fig.
![1735524339830](https://github.com/user-attachments/assets/77ad8723-d6ee-4f95-8280-46f05514657c)

**Comment by Leon1207** [comment]
Hi, I would like to see the test command you use.
And have you changed the following setting mentioned in readme?
![image](https://github.com/user-attachments/assets/624eb253-0644-450e-921a-e91cf4ee0d16)

**Comment by 1301358882** [comment]
> Hi, I would like to see the test command you use. And have you changed the following setting mentioned in readme? ![image](https://private-user-images.githubusercontent.com/60539679/399207235-624eb253-0644-450e-921a-e91cf4ee0d16.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU1Mjk4NDUsIm5iZiI6MTczNTUyOTU0NSwicGF0aCI6Ii82MDUzOTY3OS8zOTkyMDcyMzUtNjI0ZWIyNTMtMDY0NC00NTBlLTkyMWEtZTkxY2Y0ZWUwZDE2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjMwVDAzMzIyNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc0NzQ2OTYzYTBhNTkzZjgzYzE2MGIwYWE5ZDMxOGMzYTJlMDFjMWJjYjVkNTliNDkzOWM4YTVjYTdjOTZkNzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.WxsYl5QL-Inz05Jxe7fCxiNRBuG0Cu_As3zYcby5dm8)

![1735540278561](https://github.com/user-attachments/assets/2a73c42a-6a98-4848-ab6a-dfd0cb80c543)
![1735540314723](https://github.com/user-attachments/assets/565036cd-e25e-45e5-a750-db4879195a56)
The test command I used is shown in the image, and the type part has also been modified.

**Comment by Leon1207** [comment]
Hi! Thanks for your sharing.
I have noticed that you changed the config file after running the training script (already generated the cache file in 'exp' dir), so you need to change it in the cache file, not in the config file. Please refer to:
![image](https://github.com/user-attachments/assets/c07d4601-3615-4bae-be5a-3829f28e299b)

![image](https://github.com/user-attachments/assets/b9d8f9b7-a7d9-464b-a80f-631cca7e7edf)
